### PR TITLE
Fix app installation

### PR DESCRIPTION
**Pull request by rachellerathbone**
I tried installing the app and got this lovely error:

Screen Shot 2023-02-03 at 9 57 39 am

Turns out we neglected to add a route to our connect descriptor so made some changes to modify what we had and make sure the install works and views still render.

Screen Shot 2023-02-03 at 10 34 58 am

Screen Shot 2023-02-03 at 10 36 28 am

Screen Shot 2023-02-03 at 10 36 22 am

### CODE REVIEW

File: src/config/env.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { Router } from 'express';
 import { connectAppDescriptor } from '../config';
-import baseUrl from '../utils';
+import { baseUrl } from '../utils';
```
Comment (rachellerathbone) [inline src/config/env.ts:3]: Had to change this as the export was modified in a previous PR and no longer working.

File: src/config/index.ts [hunk 2]
```diff
@@ -1,52 +1,59 @@
-type Props = {
+type ConnectAppDescriptorProps = {
```
Comment (rachellerathbone) [inline src/config/index.ts:1]: Renamed for clarity.

File: src/utils/index.ts [hunk 3]
```diff
@@ -6,9 +6,11 @@
...
 export const baseUrl = async (): Promise<string> => {
     const response = await fetch('http://tunnel:4040/api/tunnels');
+
     if (!response.ok) {
         throw new Error(`ngrok tunnel error: ${response}`);
     }
+
     const data = await response.json();
-    return data.tunnels[0].public_url;
+    return data.tunnels[1].public_url;
```
Comment (rachellerathbone) [inline src/utils/index.ts:15]: Needed to modify this as we were passing in the url with http protocol. Installation of the app requires https

**Review by mboudreau — APPROVED** [review]

File: src/config/index.ts (hunk 2 above)
Comment (rachellerathbone) [reply src/config/index.ts:1]: Do we even want to include 'Props' in this? Doesn't really feel like a TS naming convention to me. Feels more 'Reacty'

File: src/routes/atlassian-connect.ts [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import { Request, Response } from "express";
+import { connectAppDescriptor } from "../config";
+import { baseUrl } from "../utils";
+
+export const ConnectDescriptorGet = async (_: Request, res: Response): Promise<void> => {
```
Comment (krazziekay) [inline src/routes/atlassian-connect.ts:5]: We already have a route to get the connect app descriptor, `/config` without any query parameter

**Pull request merged**